src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/NewAzurePolicyDefinition.cs

```diff
@@ -131,5 +139,17 @@ private JToken GetPolicyRuleObject()
                 ? JToken.FromObject(FileUtilities.DataStore.ReadFileAsText(policyFilePath))
                 : JToken.FromObject(this.Policy);
         }
+
+        /// <summary>
+        /// Gets the parameters object
+        /// </summary>
+        private JToken GetParametersObject()
+        {
+            string parametersFilePath = this.TryResolvePath(this.Parameters);
+
+            return File.Exists(parametersFilePath)
+                ? JToken.FromObject(FileUtilities.DataStore.ReadFileAsText(parametersFilePath))
+                : JToken.FromObject(this.Policy);
+        }
     }
```

Comment:
You can add scenario tests once you have addressed the PR comments.
Also get this PR reviewed by vivsrini.